setup.cfg

```diff
@@ -0,0 +1,5 @@
+[bdist_wheel]
+universal = 1
+
+[metadata]
+license_file = LICENSE
```

Comment:
This is to ensure the license file is included in the wheel files. Cf. https://wheel.readthedocs.io/en/stable/#including-the-license-in-the-generated-wheel-file.